app/views/report/_form_columns.html.haml

```diff
@@ -26,6 +26,13 @@
       = render :partial => "column_lists"
     %strong
       = _('* Caution: Changing these fields will clear all selected values below them!')
+    - if @edit[:new][:model] && @edit[:new][:model].starts_with?('Chargeback')
+      - unavailable_fields = unavailable_fields_for_model
+      - if unavailable_fields
+        - report_base = @edit[:new][:model].titleize.split(' ').last.pluralize
```

Comment:
Translation missing here, probably `ui_lookup` but not sure.

Reply:
Here I don't display anything so I don't think so. I think I miss `ui_lookup` later.

Reply:
You're displaying `report_base` a few lines below.

Reply:
Yes, this is what i meant by "later".